docs/ja/about-appium/intro.md

```diff
@@ -0,0 +1,138 @@
+## Appium 入門
+
+Appium は、iOS モバイル、Android モバイル、Windows デスクトッププラットフォーム上の
+ネイティブ、モバイル Web、ハイブリッドアプリケーションを自動化するためのオープンソースツールです。
+**ネイティブアプリ**とは、iOS、Android、または Windows SDK を使用して開発されたアプリのことです。
+**モバイル Web アプリ**は、モバイルブラウザを使用してアクセスする Web アプリです (Appium は、iOS では Safari、Android では Chrome または内蔵の Browser アプリをサポートしています)。
+**ハイブリッドアプリ**は、Web コンテンツとのインタラクションを可能にするネイティブコントロールである「Web ビュー」のラッパーを持っています。
+[Apache Cordova](https://cordova.apache.org) や [Phonegap](http://phonegap.com/) のようなプロジェクトでは、
+Web 技術を使用してアプリを簡単に構築することができ、
+それをネイティブのラッパーにバンドルしてハイブリッドアプリを作成することができます。
+
+重要なことは、Appium は「クロスプラットフォーム」であるということです。
+つまり、同じ API を使用して、複数のプラットフォーム (iOS、Android、Windows) に対してテストを書くことができます。
+これにより、iOS、Android、Windows のテストスイート間でコードを再利用することができます。
+
+Appium がそのプラットフォームを「サポート」することの意味、
+および自動化モダリティについての具体的な情報は、
+[プラットフォームサポートドキュメント](/docs/ja/about-appium/platform-support.md)を参照してください。
+
+### Appium の理念
+
+Appium は、次の4つの信条によって概説された理念に従って、
+モバイルオートメーションのニーズを満たすように設計されました。
+
+1. アプリを自動化するために、
+アプリを再コンパイルしたり、修正したりする必要はありません
+2. テストを書いて実行するために、
+特定の言語やフレームワークに縛られるべきではありません
+3. モバイルオートメーションフレームワークは、
+自動化 API に関しては、車輪の再発明をしてはいけません
+4. モバイルオートメーションフレームワークは、名前だけでなく、
+精神的にも実践的にもオープンソースでなければなりません!
+
+### Appium のデザイン
+
+では、Appium プロジェクトの構造はどのようにしてこの理念を実現しているのでしょうか？
+ベンダーが提供する自動化フレームワークをアンダーグラウンドで使用することで、要件 1 を満たしています。
+この方法では、Appium 固有のコードやサードパーティのコードやフレームワークをアプリにコンパイルする必要はありません。
+つまり、**出荷するアプリと同じアプリをテストしていることになります**。
+使用しているベンダー提供のフレームワークは以下の通りです。
+
+* iOS 9.3 以上: Apple の [XCUITest](https://developer.apple.com/reference/xctest)
+* iOS 9.3 以下: Apple の [UIAutomation](https://web.archive.org/web/20160904214108/https://developer.apple.com/library/ios/documentation/DeveloperTools/Reference/UIAutomationRef/)
+* Android 4.3 以上: Google の [UiAutomator/UiAutomator2](https://developer.android.com/training/testing/ui-automator)
+* Windows: Microsoft の [WinAppDriver](http://github.com/microsoft/winappdriver)
+
+ベンダーが提供するフレームワークを
+[WebDriver](http://docs.seleniumhq.org/projects/webdriver/) API という1つの API でラップすることで要件2を満たしています。
+WebDriver (別名 "Selenium WebDriver") は、([JSON Wire Protocol](https://w3c.github.io/webdriver/webdriver-spec.html) として知られる) 
+クライアント - サーバプロトコルを指定します。
+このクライアント - サーバアーキテクチャを考えると、任意の言語で書かれたクライアントを使用して、
+適切な HTTP リクエストをサーバに送信することができます。
+既に[あらゆる一般的なプログラミング言語で書かれたクライアント](http://appium.io/downloads)
+が存在します。
+これはまた、好きなテストランナーやテストフレームワークを
+自由に使用できることを意味します。
+言い換えれば、Appium と WebDriver クライアントは技術的には「テストフレームワーク」ではなく、
+「自動化ライブラリ」です。
+テスト環境を
+好きなように管理することができます。
+
+同様に要件3を満たしています。
+WebDriver は、Web ブラウザを自動化するためのデファクトスタンダードとなっており、
+[W3C のワーキングドラフト](https://dvcs.w3.org/hg/webdriver/raw-file/tip/webdriver-spec.html)です。
+なぜモバイル用に全く違うことをするのでしょうか?
+その代わりに、モバイルの自動化に役立つ API メソッドを追加して
+[プロトコルを拡張](https://github.com/SeleniumHQ/mobile-spec/blob/master/spec-draft.md)しました。
+
+要件4が与えられたものであることは明らかなはずです。
```

Comment:
Which is proper for your intention?
A space around alphabet or number like "要件 1 " or no spaces like "要件2", "要件3" or "要件4". Either way is fine, but it would be nice to keep consistency.

```suggestion
要件 4 が与えられたものであることは明らかなはずです。
```

Reply:
@KazuCocoa 
The following method will be used.
- A space around alphabet or number